### ISSUE TITLE
Windows has no crontab, so no support for 'remind-me'?

### ISSUE THREAD

**Issue by Slapbox** [zeke/remind-me#60]
Is that accurate? Is there a plan to support Windows?

Thanks!

**Comment by zeke** [comment]
Yeah, that's correct. There are other ways to do the recurring checks besides cron. In theory we could have a pure javascript thing that just uses `setInterval`. I am not actively using or maintaining this library, but would be willing to help/advise out if you want to give that a try.
